frontend/src/router/index.js

```diff
@@ -50,8 +51,16 @@ const routes = [
     }
   },
   {
-    path: '/evaluation',
-    name: 'evaluation',
+    path: '/evaluations',
```

Comment:
Da du den Pfad geändert hast. Kannst du in `Leaderbord.vue` auch den Pfad noch anpassen von dem "Start new evaluation" Button 😇